fab/fabfile.py

```diff
@@ -1045,6 +1044,18 @@ def services_restart():
     _supervisor_command('start all')
 
 
+@roles(ROLES_DJANGO)
+@serial
```

Comment:
this is the default, but it's very easy to do a pass on this file and try to parallelize everything, so i'm explicitly marking as serial